tests/integration/modules/test_cmdmod.py

```diff
@@ -70,7 +70,7 @@ def test_run(self):
         self.assertEqual(self.run_function('cmd.run',
                          ['echo {{grains.id}} | awk "{print $1}"'],
                          template='jinja',
-                         python_shell=True), 'minion')
+                         python_shell=True), 'master')
```

Comment:
We shouldn't flip to `master`... We should handle the flip, if needed in the function being called?
Or maybe name it `master_minion`?

This way we know better what's being targeted?

Reply:
I'm not sure I understand the question? It's not changing the target, it's just the MasterMinion's id grain is `master`, instead of `minion`. It may be that a different grain would be a better choice here :suspect: 

Do you have any recommendations?

Reply:
`master_minion`?

Reply:
Then we'd have to change the ID in the `master` config file as well. Is that something that we want to do?

Reply:
i.e. this line:

https://github.com/saltstack/salt/blob/develop/tests/integration/files/conf/master#L3

Reply:
We're copying the options from a loaded dictionary, we would change the `id` there.
During this process do we ever load options from the config file? If so, we might want to create a temporary one with the changed ID and point salt to that one...

Reply:
Ah yeah, that would work.

For the latter, I'm not sure. I'll see about setting the ID before passing the opts in.

Reply:
Done 👍